guide/src/python-from-rust/calling-existing-code.md

```diff
@@ -24,9 +23,16 @@ fn main() -> PyResult<()> {
 }
 ```
 
-## Want to run just an expression? Then use `eval_bound`.
+[`PyModule::import`] introduces an overhead each time it's called. To avoid this in functions that are called multiple times,
+using a [`GILOnceCell`]({{#PYO3_DOCS_URL}}/pyo3/sync/struct.GILOnceCell.html) is recommended. Specifically, for importing types,
+[`GILOnceCell::import`]({{#PYO3_DOCS_URL}}/pyo3/sync/struct.GILOnceCell.html#method.import) is provided
+(check out the [example]({{#PYO3_DOCS_URL}}/pyo3/sync/struct.GILOnceCell.html#example-using-giloncecell-to-avoid-the-overhead-of-importing-a-class-multiple-times)).
```

Comment:
Sure, just made those changes in https://github.com/PyO3/pyo3/pull/4542/commits/46b18cc0a56a52987dc5c009c0a68f809fbacda8.

However, since I changed the implementation from `PyType` to `T: PyTypeCheck`, this _does_ work:

```rust
#[pyfunction]
fn sum<'py>(py: Python<'py>, l: Py<PyList>) -> PyResult<Bound<'py, PyAny>> {
    static SUM: GILOnceCell<Py<PyAny>> = GILOnceCell::new();
    SUM.import(py, "builtins", "sum")?
        .call1((l,))
}
```

I guess this raises some questions. Is there any benefit in storing functions such as `sum` in the `GILOnceCell`, too? If not, should we prevent it (maybe by changing it back to `PyType`)? And if yes, should we be explicit in the docs that this is not only for importing classes?